apps/native/app/src/screens/app-lock/app-lock.tsx

```diff
@@ -147,6 +148,7 @@ export const AppLockScreen: NavigationFunctionComponent<{
           .logout()
           .then(() => {
             preferencesStore.setState({ hasOnboardedPinCode: false })
+            preferencesStore.setState({ pinTries: 0 })
```

Comment:
Logout will now reset preferencesStore so both of these are redundant.

Reply:
Great point, had already forgotten about that 😂